services/Cargo.toml

```diff
@@ -15,6 +15,7 @@ xml = ["quick-xml"]
 postgres = ["postgres-types", "bb8-postgres"]
 odm = ["pro"]
 gfbio = ["postgres", "geoengine-datatypes/postgres"]
+nfdi = []
```

Comment:
I guess we should merge gfbio and nfdi as a flag. I suggest both is nfdi now.

Reply:
c5fa0fb1b47bcbd79d7efced816f5f360cbc0d39